dev/src/components/NavBar.tsx

```diff
@@ -1,49 +1,47 @@
 import React from "react"
-import { useAppState } from "../overmind"
+import { useActions, useAppState } from "../overmind"
 import { Link } from "react-router-dom"
-import NavBarFooter from "./navbar/NavBarFooter"
-import NavBarCourse from "./navbar/NavBarCourse"
-import { isEnrolled, isVisible } from "../Helpers"
-
+import NavFavorites from "./NavFavorites"
+import NavBarUser from "./navbar/NavBarUser"
 
 //TODO Review the NavBar behaviour.
```

Comment:
Should this be removed now, or made into an issue with more specific details?

Reply:
I believe the issue that TODO refers to has been solved a while back. Removed.